bundles/org.openhab.binding.irobot/README.md

```diff
@@ -0,0 +1,154 @@
+# iRobot binding
+
+This binding provides for integration of products by iRobot company (http://www.irobot.com/). It is currently developed to support Roomba 900
+series robotic vacuum cleaner with built-in Wi-Fi module. The binding interfaces to the robot directly without any need for a dedicated MQTT server.
+
+The development starts with an abandoned draft by hkunh42 (http://github.com/hkuhn42/openhab2.roomba) and heavily uses the following
+projects as a reference:
+
+- Roomba980-Python by Nick Waterton (http://github.com/NickWaterton/Roomba980-Python)
+- Dorita980 by Facu ZAK (https://github.com/koalazak/dorita980)
+
+## Supported Things
+
+- iRobot Roomba robotic vacuum cleaner (https://www.irobot.com/roomba). The binding has been developed and tested with Roomba 930.
+- iRobot Braava has also been reported to work except Braava-specific cycles and automatic password retrieval.
```

Comment:
No, sorry. In fact Braava works by being manually added as Roomba. Of course this implies that Braava-specific functionality is not supported. Unfortunately i cannot easily add it now because i don't own Braava and i don't have its identification details.
 I could contact its user and ask him to retrieve some logs, but it will mean further development, it will take more time. Can we proceed without it ?
 Or i can remove Braava mentioning if you think it's inappropriate.

Reply:
You could mention the Roomba ThingTypeID and that it's the same for Braava with caveat.

Reply:
Updated the description by noting this explicitly, hope you'll like it.